commands/migrate.go

```diff
@@ -59,6 +59,7 @@ func (m *migrateFnCmd) migrate(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	fmt.Println(oldFF)
```

Comment:
Are you planning to remove it later?
If so, mark it TODO so it doesnt get missed later :)